src/leaderboard.js

```diff
@@ -0,0 +1,90 @@
+import axios from 'axios';
+
+let gameID;
+const body = document.querySelector('body');
+
+async function createGame(name) {
+  const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+  try {
+    const response = await axios.post(baseURL, { name });
+    [, , , gameID] = response.data.result.split(' ');
+
+    return gameID;
+  } catch (error) {
+    const h2 = document.createElement('h2');
+    h2.innerText = '404 error hapend';
+    body.appendChild(h2);
+    throw error;
+  }
+}
+
+async function getScores() {
+  try {
+    const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameID}/scores/`;
+    const response = await axios.get(url);
+    const scores = response.data.result;
+    const list = document.querySelector('.name-num-list');
+    list.innerHTML = '';
+    scores.forEach((person) => {
+      const listItem = document.createElement('li');
+      listItem.classList.add('list-items');
+      listItem.innerHTML = `<span class="span-name">${person.user}:</span>  <span class="span-number">${person.score}</span>`;
+      list.appendChild(listItem);
+    });
+  } catch (error) {
+    const h2 = document.createElement('h2');
+    h2.innerText = '404 error hapend';
+    body.appendChild(h2);
+  }
+}
```

Comment:
- [ ] [REQUIRED] Let's remember to ensure the usage of **arrow functions** as stated by the project requirement. This is a regular async function declaration. Arrow functions have a different syntax, using the => arrow notation.
Kindly fix this 👍🏽